src/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.py

```diff
@@ -271,7 +274,9 @@ def _start_cinema4d_client(self) -> None:
         # XXX: on linux we need to run the c4d env setup script first, this env
         # var allows us to use a wrapper around Commandline. Ideally a conda env
         # does this for us.
-        c4d_exe_env = os.getenv("DEADLINE_CINEMA4D_EXE", "")
+        # On Linux this should be a path similar to this: /opt/maxon/cinema4dr2024.200/bin/c4d
+        # On Windows it should be a path similar to this: "C:\Program Files\Maxon Cinema 4D R26\Commandline.exe"
+        c4d_exe_env = os.getenv("COMMANDLINE_EXECUTABLE", "")
```

Comment:
This variable name is very generic, and just from the name I would guess it's an alias for "cmd.exe" or another command-line shell. Maybe name it `CINEMA4D_COMMANDLINE_EXECUTABLE` to make it clear it's for Cinema 4D?